schutzbot/Jenkinsfile

```diff
@@ -789,6 +785,11 @@ void run_tests(test_type) {
             label: "Base tests",
             script: "/usr/libexec/tests/osbuild-composer/base_tests.sh"
         )
+
+        sh (
+            label: "Regression tests",
+            script: "/usr/libexec/tests/osbuild-composer/regression.sh"
+        )
```

Comment:
We're in the process of deprecating Jenkins. Also add the script to .gitlab-ci.yml Base section.